frontend/apps/reader/modules/readertoc.lua

```diff
@@ -582,6 +583,20 @@ function ReaderToc:onShowToc()
         end
     end
 
+    function toc_menu:onMenuHold(item)
+        print("toc_menu:onMenuHold", item)
+        print(require("dump")(item))
+        local textviewer = TextViewer:new{
+            title = _("ToC entry"),
+            text = item.text,
+            lang = nil, --- @todo: Get doc's lang?
```

Comment:
Yep:
`lang = self.ui.document:getProps().language`
(test with printf we actually get it)